backend/src/main/java/atlantbh/restaurants/repositories/MenuItemRepository.java

```diff
@@ -0,0 +1,20 @@
+package atlantbh.restaurants.repositories;
+
+import atlantbh.restaurants.models.MenuItem;
+import atlantbh.restaurants.models.filters.MenuItemFilterBuilder;
+import atlantbh.restaurants.models.sortkeys.MenuItemSortKeys;
+import org.hibernate.Criteria;
+import org.hibernate.criterion.Restrictions;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class MenuItemRepository extends BaseRepositoryImpl<MenuItem, MenuItemSortKeys, MenuItemFilterBuilder> {
+    public List<MenuItem> findByMenu(Long menu) {
+        Criteria criteria = getBaseCriteria();
+        criteria.createAlias("menu", "menu");
+        criteria.add(Restrictions.eq("menu.id", menu));
```

Comment:
no need for alias when you fetch id from joined entity..
```return getBaseCriteria()
                      .add(Restrictions.eq("menu.id", menu))
                      .list();```

Reply:
Removed obsolete method in 0cd4debab433260ab6a8267aa0b72b31db581edb